src/ontology/mondo-edit.obo

```diff
@@ -420307,6 +420309,73 @@ intersection_of: MONDO:0001328 ! thyroid hormone resistance syndrome
 intersection_of: disease_has_basis_in_dysfunction_of http://identifiers.org/hgnc/11799 ! THRB
 relationship: disease_has_basis_in_dysfunction_of http://identifiers.org/hgnc/11799 {source="Orphanet:566243"} ! THRB
 
+[Term]
+id: MONDO:0034846
+name: primary desmosis coli
+def: "A rare intestinal disease characterized by congenital partial or complete lack of the collagen mesh network in the intestinal wall, resulting in hypoperistalsis or aperistalsis. The enteric nervous system is normal or near-normal in the affected areas, although hypo- and dysganglionosis may be found in some proximal segments of the colon and/or small bowel. Patients present with chronic intractable slow transit constipation." [Orphanet:565641]
+subset: ordo_disease
+xref: ICD10:K59.9 {source="Orphanet:565641/ntbt"}
+xref: Orphanet:565641 {source="MONDO:equivalentTo"}
+is_a: MONDO:0005020 {source="Orphanet:565641"} ! intestinal disease
+
+[Term]
+id: MONDO:0034849
+name: laminin subunit alpha 2-related limb-girdle muscular dystrophy R23
+subset: ordo_disease
+xref: Orphanet:565837 {source="MONDO:equivalentTo"}
+is_a: MONDO:0015152 {source="Orphanet:565837"} ! autosomal recessive limb-girdle muscular dystrophy
+is_a: MONDO:0016149 {source="Orphanet:565837"} ! qualitative or quantitative defects of merosin
+
+[Term]
+id: MONDO:0034972
+name: NLRC4-related familial cold autoinflammatory syndrome
+subset: ordo_disease
+xref: OMIM:616115 {source="Orphanet:576349"}
+xref: Orphanet:576349 {source="MONDO:equivalentTo"}
+is_a: MONDO:0000001 {source="Orphanet:576349"} ! disease or disorder
+is_a: MONDO:0017370 {source="Orphanet:576349"} ! autoinflammatory syndrome with skin involvement
+is_a: MONDO:0017953 {source="Orphanet:576349"} ! hereditary periodic fever syndrome
+relationship: disease_has_basis_in_dysfunction_of http://identifiers.org/hgnc/16412 {source="Orphanet:576349"} ! NLRC4
+
+[Term]
+id: MONDO:0034976
```

Comment:
need an iatrogenic DP - can you do @nicolevasilevsky - but axioms can be added later